### PR TITLE
DOC: RangeIndex as default index

### PR DESCRIPTION
**Pull request by max-sixty**
Small correction to DataFrame docstring

### CODE REVIEW

File: pandas/core/frame.py [hunk 1]
```diff
@@ -251,11 +251,11 @@ class DataFrame(NDFrame):
     data : numpy ndarray (structured or homogeneous), dict, or DataFrame
         Dict can contain Series, arrays, constants, or list-like objects
     index : Index or array-like
-        Index to use for resulting frame. Will default to np.arange(n) if
+        Index to use for resulting frame. Will default to RangeIndex if
         no indexing information part of input data and no index provided
     columns : Index or array-like
         Column labels to use for resulting frame. Will default to
-        np.arange(n) if no column labels are provided
+        RangeIndex if no column labels are provided
```
Comment (jreback) [inline pandas/core/frame.py]: can u check Series doc string as well

Comment (maxim-lian) [reply pandas/core/frame.py]: Series is good already

File: pandas/core/frame.py [hunk 2]
```diff
@@ -251,11 +251,11 @@ class DataFrame(NDFrame):
     data : numpy ndarray (structured or homogeneous), dict, or DataFrame
         Dict can contain Series, arrays, constants, or list-like objects
     index : Index or array-like
-        Index to use for resulting frame. Will default to np.arange(n) if
+        Index to use for resulting frame. Will default to RangeIndex if
```
[the remark this answers is not in the document]
Comment (maxim-lian) [reply pandas/core/frame.py]: Let's do `RangeIndex (0, 1, 2, ..., n)`? 

Some may not know `arange(n)` either

[the remark this answers is not in the document]
Comment (jreback) [reply pandas/core/frame.py]: @maxim-lian your suggestion looks good. pls ping when ready. (and update Series to similar)

**Comment by maxim-lian** [comment]
Unrelated test failures: https://travis-ci.org/maxim-lian/pandas/jobs/344592059

**Comment by jreback** [comment]
thanks!

File: pandas/core/frame.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jorisvandenbossche) [reply pandas/core/frame.py]: that was indeed a good suggestion!

**Pull request merged**